File: api/src/main/java/com/fosscut/api/type/AirflowDAGLogsDTO.java
package com.fosscut.api.type;

public class AirflowDAGLogsDTO {

    private String status;
    private String logs;

    public AirflowDAGLogsDTO() {}

    public AirflowDAGLogsDTO(String status, String logs) {
        this.status = status;
        this.logs = logs;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getLogs() {
        return logs;
    }

    public void setLogs(String logs) {
        this.logs = logs;
    }

}

File: gui/client/src/type/airflow/AirflowDAGLogsDTO.ts
import { z } from 'zod';

const AirflowDAGLogsDTO = z.object({
  status: z.string().nullable(),
  logs: z.string()
});

type AirflowDAGLogsDTO = z.infer<typeof AirflowDAGLogsDTO>;

export default AirflowDAGLogsDTO;
